csrc/compute_at_map.cpp

```diff
@@ -745,6 +745,31 @@ void IterDomainGraph::build(Fusion* fusion) {
     }
   };
 
+  auto compare_with_id_model = [](const DisjointSets<IterDomain*>& nodes,
+                                  IdMappingMode mode) {
+    if (!GpuLower::hasCurrent() || !GpuLower::current()->hasIdModel()) {
+      return;
+    }
+
+    const ValGraph& graph = GpuLower::current()->idModel().idGraph(mode);
+    for (const auto& ca_map_group : nodes.disjointSets()) {
+      IterDomain* first_id = nullptr;
+      for (const auto& id : *ca_map_group) {
+        NVF_ERROR(graph.hasGroup(id), "Not found in IdModel:", id->toString());
+        if (first_id == nullptr) {
+          first_id = id;
+        } else {
+          NVF_ERROR(
```

Comment:
This assertion got triggered only with the legacy swizzle tests. That's expected as IdModel doesn't do anything for the legacy swizzle ops.

CC: @zasdfgbnm